src/dbnode/environment/config.go

```diff
@@ -25,30 +25,32 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/m3db/m3/src/dbnode/kvconfig"
+
```

Comment:
nit: These should be grouped with the other imports, suprising linter is not catching these exceptions anymore.